os/customizing-sshd.md

```diff
@@ -23,11 +23,11 @@ storage:
           # Use most defaults for sshd configuration.
           UsePrivilegeSeparation sandbox
           Subsystem sftp internal-sftp
+          UseDNS no
 
           PermitRootLogin no
           AllowUsers core
-          PasswordAuthentication no
-          ChallengeResponseAuthentication no
```

Comment:
I feel like we've been through this before.  My system's `sshd_config` says about `UsePAM`:

> If you just want the PAM account and session checks to run without
> PAM authentication, then enable this but set PasswordAuthentication
> and ChallengeResponseAuthentication to 'no'.